projects.yml

```diff
@@ -340,6 +340,12 @@ hato:
   categories: [HTTP Clients]
   platforms: [clj]
 
+java-http-clj:
```

Comment:
Can you use underscores in the key, instead of dashes?

Reply:
My apologies, I should have spotted that! I've have now fixed it.